.gitlab/source_test/golang_deps_diff.yml

```diff
@@ -21,6 +21,8 @@ golang_deps_commenter:
   image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/pr-commenter:2
   tags: ["arch:amd64"]
   needs: ["golang_deps_diff"]
+  rules:
+    - if: '$CI_PIPELINE_SOURCE == "merge_request_event"'
```

Comment:
Can this really work? AFAIK none of the pipelines we run have the `merge_request_event`, since we don't open merge requests in Gitlab.

Reply:
You are probably right (this is not tested yet) and if I find a correct condition I was wondering if it should take place in the gitlab-ci.xml file